openedx/features/edly/middleware.py

```diff
@@ -26,7 +26,10 @@ def process_request(self, request):
 
         user_is_authenticated = request.user.is_authenticated
         user_is_superuser = request.user.is_superuser
-        if user_is_authenticated and not user_is_superuser and not user_has_edly_organization_access(request):
```

Comment:
@asadmanzoor93 we need to simplify checks
```
if request.user.is_superuser or request.user.is_staff:
    pass
if not user_has_edly_organization_access(request):
.......................
```

Reply:
Feedback implemented